snakemake-slurm.md

```diff
@@ -18,37 +18,54 @@ Here's an example cluster configuration file:
 ```
 # cluster_config.yml - cluster configuration
 __default__:
-    account: ACCOUNT
-    partition: PARTITION
-    time: 01:00:00 # time limit for each job
-    nodes: 1
-    ntasks-per-node: 14 #Request n cores be allocated per node.
-    chdir: /directory/to/change/to
-    output: a_name_for_my_job-%j.out
-    error: a_name_for_my_job-%j.err
+    account: ctbrowngrp
+    partition: bml
+    time: 00:05:00 # time limit for each job (5 min)
```

Comment:
we may want to use an example that includes the number of days e.g. `01-00:05:00 # time limit for each job fmt = dd-hh:mm:ss`